apps/examples/iotjs/setjmp/Make.defs

```diff
@@ -17,5 +17,5 @@
 ###########################################################################
 
 ifeq ($(CONFIG_EXAMPLES_SETJMP_TEST),y)
-CONFIGURED_APPS += examples/setjmp
+CONFIGURED_APPS += examples/iotjs/setjmp
```

Comment:
As far as I know, it's an example of https://github.com/Samsung/TizenRT/blob/master/os/arch/arm/src/common/setjmp.S#L93, not iotjs.